### ISSUE TITLE
Documentation of java.lang.BigDecimal does not match serialization

### ISSUE THREAD

**Issue by mjiderhamn** [teamcarma/swagger-jaxrs-doclet#87]
A `java.lang.BigDecimal` that is serialized to JSON using Jersey 2.17/Jackson 2.3.2 will be represented as only the numeric value.
swagger-jaxrs-doclet however will produce `BigDecimal` and `BigInteger` types, that make the documentation both confusing and incorrect.
```
...
        "foo" : {
          "type" : "BigDecimal"
        },
...
    "BigInteger" : {
      "id" : "BigInteger",
      "properties" : {
        "signum" : {
          "type" : "integer",
          "format" : "int32"
        },
        "mag" : {
          "type" : "int[]"
        },
        "bit" : {
          "type" : "BigInteger"
        },
        "probablePrime" : {
          "type" : "boolean"
        }
      }
    },
    "BigDecimal" : {
      "id" : "BigDecimal",
      "properties" : {
        "intVal" : {
          "type" : "BigInteger"
        },
        "scale" : {
          "type" : "integer",
          "format" : "int32"
        }
      }
    },
```
I'm assuming the same goes for `java.lang.BigInteger`

**Comment by dwalden** [comment]
+1

**Comment by conorroche** [comment]
I have now added support for big decimal and big integer. I treat big decimal as a double and big integer as a long.

**Comment by conorroche** [comment]
closing as fixed for 1.0.5 release
